tests/examples/knowledge_tuning/conftest.py

```diff
@@ -0,0 +1,32 @@
+"""Pytest configuration and fixtures for knowledge-tuning tests."""
```

Comment:
@andrewdonheiser looks like @tarun-etikala is adding similar pytests for knowledge_tuning example notebooks akin to what you proposed under the push tests PR - https://github.com/red-hat-data-services/red-hat-ai-examples/pull/16/files

Did you both want to reconcile these specific changes into a single PR and reduce the scope in one of these PRs ?